tests/ExceptionHandlerTest.php

```diff
@@ -156,13 +155,11 @@ public function testInvalidFormatterClass()
 
         $property->setValue($handler, $config);
 
-        $this->setExpectedException(
-            \InvalidArgumentException::class,
-            sprintf(
-                "% is not a valid formatter class.",
-                get_class($formatter)
-            )
-        );
```

Comment:
setExpectedException is no longer supported in phpUnit 8, so replaced by equivalent code